.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
please, no husky hooks. They cause more pain than it's worth. Having a CI check is sufficient to enforce the markdown rules.

Reply:
@gr2m I can remove it

Reply:
@gr2m, is the pain from the tooling (husky, lint-staged) or pre-commit hooks in general?

Isn't it good to catch lint errors locally rather than CI? Faster feedback loop